quesma/quesma/functionality/bulk/bulk.go

```diff
@@ -54,6 +54,7 @@ type (
 		Result      string             `json:"result,omitempty"`
 		Status      int                `json:"status"`
 		Error       any                `json:"error,omitempty"`
+		Type        string             `json:"_type"`
```

Comment:
shouldn't we add `omitempty` here?

Reply:
We always set it, so it's never empty. `Result`,`Error` is `omitempty` because we only fill one of them (`Result` for successes, `Error` for failures)

Reply:
I vaguely recall that it's been deprecated and no longer used in `8.x` (to be verified), but as long as it doesn't present any issues for any API clients - I'm be fine with that.

Reply:
Added a comment here that some clients require this field.